src/treeView.ts

```diff
@@ -184,7 +188,24 @@ export class TreeDataProvider implements vscode.TreeDataProvider<TreeItem> {
       let functionalTestsItem = this.data.find((item) => item.label && item.label.toString().startsWith("Functional"));
       let buidUseCaseItem = this.data.find((item) => item.label && item.label.toString().startsWith("Build"));
       let selectVariantItem = this.data.find((item) => item.label && item.label.toString().startsWith("Select variant"));
-
+      let checkItem = this.data.find((item) => item.label && item.label.toString().startsWith("Run Guideline Enforcer"));
+
+      this.data.forEach((item) => {
```

Comment:
Why do you need all this forEach logic ? For other items we just do `this.data.find` directly